webapp/player-project/public/locales/fr/translation.json

```
@@ -0,0 +1,12 @@
+{
+  "notifications": {
+    "streamingFinished": "Le streaming est terminé, lecture de l'enregistrement comme solution de repli",
```

suggestion: I would go with this instead.

```suggestion
    "streamingFinished": "Le streaming est terminé, lecture de l’enregistrement final",
```